pkg/gui/commits_panel.go

```diff
@@ -586,3 +586,12 @@ func (gui *Gui) handleGotoBottomForCommitsPanel(g *gocui.Gui, v *gocui.View) err
 
 	return nil
 }
+
+func (gui *Gui) handleCopySelectedCommitMessageToClipboard() error {
+	commit := gui.getSelectedLocalCommit()
+	if commit == nil {
+		return nil
+	} else {
+		return gui.OSCommand.CopyToClipboard(commit.Name)
```

Comment:
This will work for single line commit messages but not multiline. We actually don't load a commit's message in its entirety into lazygit for performance reasons. You can use:
```
	message, err := gui.GitCommand.GetCommitMessage(commit.Sha)
	if err != nil {
		return gui.surfaceError(err)
	}
```
to get the message, and then use that in place of `commit.Name` :)